src/starterbot/StarterBot.cpp

```diff
@@ -36,6 +40,10 @@ void StarterBot::onFrame()
     // Build more supply if we are going to run out soon
     buildAdditionalSupply();
```

Comment:
The `buildAdditionalSupply()` function can be deleted and replaced with `buildNumberOfUnits()`.